docs/hello-sample.md

```diff
@@ -1,150 +1,300 @@
+- TypeScript/Node.js in the folder [/samples/Hello_Netherite_with_TypeScript](https://github.com/microsoft/durabletask-netherite/tree/main/samples/Hello_Netherite_with_TypeScript)
+- Python in the folder [/samples/Hello_Netherite_with_Python](https://github.com/microsoft/durabletask-netherite/tree/main/samples/Hello_Netherite_with_Python)
+
+In addition we provide several PowerShell scripts that enable you to setup the necessary resources on Azure as well as to build, run, and deploy the sample applications. You find them in the folder [/samples/scripts](https://github.com/microsoft/durabletask-netherite/tree/main/samples/scripts)
 
 ## Prerequisites
 
+### General Prerequisites
+
 First, make sure you have the following installed on your machine:
-1. [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1) or later. 
-You can run `dotnet --list-runtimes` to check what is installed.
-2. [Azure Functions Core Tools 3.x](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Ccsharp%2Cbash) or later.
-You can run `func --version` to check what is installed.
-3. [The Azure CLI 2.18.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) or later.
-You can run `az --version` to check what is installed.
-4. [PowerShell 7.1](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.1) or later.
-You can check what is installed with `pwsh --version`. PowerShell is easy to install or update via `dotnet tool install --global PowerShell` or `dotnet tool install --global PowerShell`, respectively.
 
-Also, you need an Azure subscription, to allocate and deploy the required resources.
+1. [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1) or later. You can run `dotnet --list-runtimes` to check what is installed.
+
+2. [Azure Functions Core Tools 3.x](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Ccsharp%2Cbash) or later. You can run `func --version` to check what is installed.
+
+3. [The Azure CLI 2.18.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) or later. You can run `az --version` to check what is installed.
+
+4. [PowerShell 7.1](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.1) or later. You can check what is installed with `pwsh --version`. PowerShell is easy to install or update via `dotnet tool install --global PowerShell` or `dotnet tool install --global PowerShell`, respectively.
+
+5. [Azurite 3.x](https://github.com/Azure/Azurite) as storage emulator for the local execution. You can run `azurite --version` to check what is installed.
+
+6. [An Azure subscription](https://azure.microsoft.com/free/search/), to allocate and deploy the required resources.
```

Comment:
Maybe you could mention here that it is possible to run the samples local-only without an Azure account?

Reply:
Added that as a remark